examples/simple.js

```diff
@@ -1,37 +1,81 @@
 // if you don't specify a html file, the sniper will generate a div with id "rootDiv"
 var app = require("biojs-vis-pca");
 
-//metaData contains different options
-var metaData = {"option1": ["Republican", "Democrat"], "option2": ["a", "b", "c"], "option3": ["x", "y", "z"]};
+//colorBy contains different color options
+var colorBy = {SampleType: ["0", "Day1", "Day2","Day3", "Day4", "Day5", "Day6", "Day7", "ESRP KD", "CTRL", "RBM47 KD"], ExperimentalSeries: ["PRJNA304414", "PRJNA304419", "PRJNA304418"]};
+
+var colors = ["DarkOrchid", "Orange", "DodgerBlue", "Blue","BlueViolet","Brown", "Deeppink", "BurlyWood","CadetBlue",
```

Comment:
OK. Thanks. How many colours should we default to?\

Can we also try doing that programmatically? ie. http://bl.ocks.org/aaizemberg/78bd3dade9593896a59d